packages/micropip/micropip/micropip.py

```diff
@@ -198,7 +198,7 @@ def do_resolve(*args):
             self.installed_packages[name] = ver
 
     def add_requirement(self, requirement: str, ctx, transaction):
-        if requirement.startswith(("http://", "https://")):
+        if requirement.endswith(".whl"):
```

Comment:
Please add a test to `packages/micropip/test_micropip.py` with just plain python (no pyodide or selenium fixture). You can use monkeypatching if needed. See example in https://github.com/iodide-project/pyodide/pull/846